extension-manifest-v3/src/pages/onboarding/index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width" />
     <meta name="theme-color" content="#000000" />
     <title>Welcome to Ghostery</title>
-    <script src="./index.js" type="module" async></script>
+    <script src="./index.js" type="module"></script>
```

Comment:
haven't you just added it?

Reply:
Leftover, I was checking if this was a couse of some Firefox problems, but it looks that it can be `async`. I'll rollback.